### DISCUSSION TITLE
Thank you for your hard work

### DISCUSSION THREAD

**Discussion by appdevnick** [OctoD/godot-gameplay-systems#52] [General]
Thanks again for doing so much to replace portions of GAS for those of us migrating from Unreal. I would love to help in any way in the future if needed.

**Comment by OctoD** [comment]
Hi thank you! At this moment I am porting to native gdextension all the core functionalities of this addon. Each functionality is going to live in a separated repo, this repo will contain only the binaries and some glue between those for v1.x.

**Comment by appdevnick** [comment]
That sounds fantastic, I look forward to seeing what you put together!
